jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js

```diff
@@ -33,7 +33,8 @@ YUI.add('service-overview', function() {
       modelUUID: React.PropTypes.string.isRequired,
       service: React.PropTypes.object.isRequired,
       serviceRelations: React.PropTypes.array.isRequired,
-      showActivePlan: React.PropTypes.func.isRequired
+      showActivePlan: React.PropTypes.func.isRequired,
+      showPlans: React.PropTypes.bool.isRequired
```

Comment:
This doesn't seem related to how URLs are parsed. The name of the PR is misleading.